lib/build-cloud/launchconfiguration.rb

```diff
@@ -80,8 +77,132 @@ def create
 
         end
 
-        launch_config = @as.configurations.new( options )
-        launch_config.save
+        unless exists?
+            @log.info( "Creating launch configuration #{@options[:id]}" )
+            launch_config = @as.configurations.new( options )
+            launch_config.save
+        else
+            # Some data jogging to get the block device mapping options into the 
```

Comment:
I wonder if we could both DRY this up and make it more future-proof by enumerating the attributes from the model (https://github.com/fog/fog-aws/blob/master/lib/fog/aws/models/auto_scaling/configuration.rb#L6-L24) rather than doing it here by hand?